scripts/data_overlap/get_group_overlap_stats.py

```
@@ -0,0 +1,91 @@
+    for data_overlap_stats in data_overlap_stats_list:
+        data_overlap_stats_key = data_overlap_stats.data_overlap_stats_key
+        light_scenario_key = data_overlap_stats_key.light_scenario_key
+        scenario_spec = light_scenario_key.scenario_spec
+        num_instances = data_overlap_stats.num_instances
+        n = data_overlap_stats_key.overlap_protocol_spec.n
+        num_overlapping_inputs = len(data_overlap_stats.instance_ids_with_overlapping_input)
+        num_overlapping_references = len(data_overlap_stats.instance_ids_with_overlapping_reference)
+        if n == 13:
+            scenario_spec_overlap_counts[scenario_spec] = (
+                num_instances,
+                num_overlapping_inputs,
+                num_overlapping_references,
+            )
+
+    group_scenario_specs_jsons = open(args.group_scenario_specs_path, "r").readlines()
+
+    group_scenario_specs_list: List = []
+    for group_scenario_specs_json in group_scenario_specs_jsons:
+        group_scenario_specs_dict = json.loads(group_scenario_specs_json)
+        group_scenario_specs_list.append(cattrs.structure(group_scenario_specs_dict, GroupScenarioSpecs))
+
+    group_overlap_stats_list: List = []
+    for group_scenario_specs in group_scenario_specs_list:
+        group = group_scenario_specs.group
+        scenario_specs = group_scenario_specs.scenario_specs
+        group_num_instances = 0
+        group_num_overlapping_inputs = 0
+        group_num_overlapping_references = 0
+        for scenario_spec in scenario_specs:
+            try:
+                num_instances, num_overlapping_inputs, num_overlapping_references = scenario_spec_overlap_counts[
+                    scenario_spec
+                ]
+                group_num_instances += num_instances
+                group_num_overlapping_inputs += num_overlapping_inputs
+                group_num_overlapping_references += num_overlapping_references
+            except Exception:
```

In general, bad style to catch a blanket Exception - can easily result in silent errors...  I don't think there should be a try/Except here...